### ISSUE TITLE
Do a real world end to end example in github

### ISSUE THREAD

**Issue by OR13** [decentralized-identity/spec.rebase.org#8]
possibly building off of https://github.com/transmute-industries/public-credential-registry-template

We should be able to show how a "did:web" and a "github account" are related... and we can use this repo to provide a real world example of that.

**Comment by oed** [comment]
Do we want to converge on a registry for credentials? Or are you more referring to doing a real world example of the end to end flow of generating a github credential?

**Comment by OR13** [comment]
@oed i meant that we should generate a real example of a rebase "linked github account credential", and I shared some tooling that could help automating those credentials.

**Comment by oed** [comment]
Makes sense. I believe Spruce and 3Box Labs has services that verify github already that we could base this on.
[Ours](https://github.com/ceramicstudio/identitylink-services) needs to use the new schema proposed in rebase. It issues a JWT credential using a `did:web`.

**Comment by OR13** [comment]
@oed  would you be willing to do a PR that adds your schema here: https://github.com/rebase-xyz/spec.rebase.xyz/tree/main/spec/openapi/schemas/common

or would you prefer to create a new schema to match your existing data model?

@wyc we probably need to start breaking some eggs to move this along, I would accept a PR that changed the existing schemas.

**Comment by oed** [comment]
I would prefer to converge on a common standard

**Comment by OR13** [comment]
@oed I interpret that as you are willing to conform to a new standard, and absorb some breaking changes in the process.

I am in the same boat.

In order to get there, we mostly need to be iterating on the JSON data models, and driving out any one off behavior associated with certain types.

I think today all we have is a GitHub example.

https://github.com/rebase-xyz/spec.rebase.xyz/tree/main/spec/openapi/schemas/common

I would approve PRs that added new schemas or that broke the existing one in some useful ways.
